pycoast/cw_agg.py

```diff
@@ -61,23 +61,35 @@ def _draw_polygon(self, draw, coordinates, **kwargs):
 
     def _draw_rectangle(self, draw, coordinates, **kwargs):
         """Draw rectangle."""
-        pen = aggdraw.Pen(kwargs['outline'])
+        pen = aggdraw.Pen(kwargs['outline'],
+                          kwargs['width'],
+                          kwargs['outline_opacity'])
```

Comment:
How did this ever work before?

Reply:
I think aggdraw have default values for these args